README.md

```diff
@@ -138,7 +138,7 @@ docker build -t copilot-metrics-viewer .
 
 ### Docker run
```

Comment:
need to update docs on how to docker run with API proxy and how to deploy with AZD up